packages/app/src/App.tsx

```diff
@@ -20,7 +20,7 @@ import {
 	TechDocsReaderPage,
 } from '@backstage/plugin-techdocs';
 import {TechDocsAddons} from '@backstage/plugin-techdocs-react';
-import {ExpandableNavigation, ReportIssue} from '@backstage/plugin-techdocs-module-addons-contrib';
+import { /*ExpandableNavigation*/ ReportIssue} from '@backstage/plugin-techdocs-module-addons-contrib';
```

Comment:
Wondering why this is commented out vs deleting?

Reply:
I commented it out because I'd prefer to put it back in at some point when/if the issue is resolved.
Considering there's no timeline on that perhaps commenting it out is supremely optimistic.

Reply:
Ok, I'm good with what ever you want to do